api/src/services/collections.ts

```diff
@@ -345,8 +348,50 @@ export class CollectionsService {
 				await this.cache.clear();
 			}
 
+			if (opts?.autoPurgeSystemCache !== false) {
+				await clearSystemCache();
+			}
+		}
+	}
+
+	/**
+	 * Update multiple collections in a single transaction
+	 */
+	async updateBatch(data: Partial<Collection>[], opts?: MutationOptions): Promise<string[]> {
+		if ((this.accountability && this.accountability.admin !== true) || !Array.isArray(data)) {
```

Comment:
`|| !Array.isArray(data)` Do we want to throw a Forbidden error for this? I think an `InvalidPayload` may be more appropriate 🤔

Reply:
Yup, refactored. Thanks! 👍